engine/src/main/java/org/destinationsol/assets/Assets.java

```diff
@@ -205,6 +208,56 @@ public static DSTexture getDSTexture(String path) {
         return getAtlasRegion(path, Texture.TextureFilter.Nearest);
     }
 
+    public static Animation<TextureAtlas.AtlasRegion> getAnimation(String texturePath) {
+        if (assetHelper.list(DSTexture.class, texturePath).size() == 0) {
+            return null;
+        }
+
+        String animationPath = texturePath + "Animation";
+        if (assetHelper.list(Json.class, animationPath).size() == 0) {
+            return null;
+        }
+
+        Texture originalTexture = getDSTexture(texturePath).getTexture();
+
+        Json animationInfoJson = getJson(animationPath);
+        JSONObject animationInfo = animationInfoJson.getJsonValue();
+        int frameWidth = animationInfo.optInt("frameWidth", 256);
+        int frameHeight = animationInfo.optInt("frameHeight", 256);
+        int frameCount = animationInfo.optInt("frameCount", 1);
+        float framesPerSecond = animationInfo.optFloat("framesPerSecond", 24);
```

Comment:
Is this going to cause problems with android, like https://github.com/MovingBlocks/DestinationSol/pull/439/commits/2480c79ed6077ffdb6daa78d40dca212d6c948c5 ? Or was that unrelated?